security-whitepaper/cloud-native-security-whitepaper.md

```diff
@@ -307,15 +307,13 @@ Hardening of infrastructure and workloads should be supported by comprehensive t
 
 ##### Static Analysis and Security Testing
 
-Static analysis of IaC, application manifests, and software code may cover linting, identifying misconfigurations and vulnerability scanning. IaC code should be subject to the same pipeline policy controls as are application workloads.
+Static analysis of IaC, application manifests, and software code provide linting, identification of misconfigurations, and vulnerabilities of the specific component and will not include context of the surrounding components in analysis.  It is important to note that individual analysis is important, but should not be the only form of analysis. IaC code should be subject to the same pipeline policy controls as are application workloads.
```

Comment:
> It is important to note that individual analysis is important, but should not be the only form of analysis.

I think that this is a bit too much of a detail for the whitepaper. This is more a consideration for the implementation of the tools rather than the operator implementing these tooling. 

> IaC code should be subject to the same pipeline policy controls as are application workloads.

I would prefer the term similar since it is not going to be exactly the same as workloads.

Reply:
```suggestion
Static analysis of IaC, application manifests, and software code provide linting, identification of misconfigurations, and vulnerabilities of the specific component and will not include context of the surrounding components in analysis.  It is important to note that individual analysis is important, but should not be the only form of analysis. IaC code should be subject to the similar pipeline policy controls as are application workloads.
```